modules/ossm-migrating-gateways-canary.adoc

```diff
@@ -0,0 +1,85 @@
+
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-gateways-canary_{context}"]
+= Gateway canary migration
+
+The gateway canary migration method is recommend for a gradual rollout. It uses multiple gateway versions with maximum control over your gateway rollout.
+
+[NOTE]
+====
+The label for the gateway namespace differs between mulitenant and cluster-wide meshes. Return to the either the "Multitenant migration guide" or the "Cluster-wide migration guide" to understand which labels to use in your specific migration case, and to understand why the `maistra.io/ignore-namespace: "true"` label is needed.
+====
+
+.Procedure
+
+. Label the gateway namespace to ensure injection from the new mesh is enabled for the namespace, ensuring to add the `maistra.io/ignore-namespace: "true"` label as well as remove `istio-injection=enabled` if needed. For example:
++
+[source,terminal]
+----
+$ oc label namespace ${APP_NAMESPACE} istio.io/rev=${ISTIO_REVISION} maistra.io/ignore-namespace="true"
+----
+
+. Deploy a canary gateway:
++
+.Example yaml for canary gateway
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: istio-ingressgateway-canary
+  namespace: istio-ingress <1>
+  spec:
+     selector:
+       matchLabels:
+         istio: ingressgateway <2>
+     template:
+       metadata:
+         annotations:
+           inject.istio.io/templates: gateway
+         labels:
+           istio: ingressgateway
+           istio.io/rev: canary <3>
+       spec:
+         containers:
+         - name: istio-proxy
+           image: auto
+----
+<1> Must be in in the same namespace as your existing gateway
+<2> Must match your existing gateway service selector.
+<3> Set to your {SMProduct} 3.0 control plane revision
+
+. Ensure that the new gateway deployment is running with the new revision and is handling requests:
++
+.. Check that pods are running and ready.
```

Comment:
```suggestion
.. Check that pods are running and in the `Ready` status.
```

I believe there must be `oc get pods` command to go with this step. Please confirm with the SME/QE.